taskcluster/ci/config.yml

```diff
@@ -34,7 +34,12 @@ workers:
             provisioner: 'mobile-{level}'
             implementation: docker-worker
             os: linux
-            worker-type: 'b-linux'
+            worker-type: b-linux
+        b-android-large:
+            provisioner: 'mobile-{level}'
+            implementation: docker-worker
+            os: linux
+            worker-type: b-linux-large
```

Comment:
This instance type is used by android-components. Using it reduces the time taken by the dependencies task from 30-40 minutes down to 20-30.

Reply:
I want to start looking into reducing the number of build variants. I believe that this will also significantly reduce the `assemble` time.